setup.cfg

```diff
@@ -40,7 +40,7 @@ where=src
 [options.extras_require]
 server =
     # These dependecies are made optional to make it easier to build extensions
-    dirac ~=7.3.0a0
+    dirac ~=7.3.5
```

Comment:
Mmm I am no expert here but this does not look correct to me.

Reply:
As far as I understand, this is the behavior we expect, no?